modules/widgets/simple-payments/form.php

```diff
@@ -96,7 +96,7 @@ class="field-currency widefat jetpack-simple-payments-form-product-currency"
 			name="<?php echo esc_attr( $this->get_field_name( 'form_product_currency' ) ); ?>">
 			<?php foreach( Jetpack_Simple_Payments_Widget::$supported_currency_list as $code => $currency ) {?>
 				<option value="<?php echo esc_attr( $code ) ?>"<?php selected( $instance['form_product_currency'], $code ); ?>>
-					<?php esc_html_e( $code . ' ' . $currency ) ?>
+					<?php echo esc_html( "$code $currency" ); ?>
```

Comment:
Replacing this here because `esc_html_e` is meant to be used when translating not for all echo.